### PR TITLE
Qpw 66

### PR DESCRIPTION
**Pull request by ashutoshqsofte**
Created New Page. Now user can be able to connect with us.

### CODE REVIEW

File: src/GlobalStyle.js
Comment (abhijeet-qsofte) [inline src/GlobalStyle.js:1]: please remove the bank space

File: src/pages/Connect.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const Connect = () => {
+  return (
+    <>
+      <h1>Ashutosh</h1>
```
Comment (abhijeet-qsofte) [inline src/pages/Connect.js]: Why do you need your name?

File: src/GlobalStyle.js
Comment (ashutoshqsofte) [reply src/GlobalStyle.js:1]: Okay sir will do

File: src/pages/Connect.js (hunk 1 above)
Comment (ashutoshqsofte) [reply src/pages/Connect.js]: I had Written to check something but it has been removed

**Pull request merged**